### PR TITLE
Pass version to presetEnv

### PR DESCRIPTION
**Pull request by bemnlam**
# ↪️ Pull Request

Fixing #5943

[This change](https://github.com/babel/babel/pull/12934/files#diff-cddb13dc00454073d19373b40529b8047e25b9141ab627eb8be482601f16eabeR301) in babel breaks Parcel if version doesn't pass into `presetEnv()`. By setting version as 7.13.0 `presetEnv()` will continue other checking, just like the logic before.

### CODE REVIEW

**Comment by devongovett** [comment]
I think this is a v1 only bug? This code path won't actually be hit in v2 because config.isSource is always true.

**Comment by nicolo-ribaudo** [comment]
Q: If that code path won't be hit, the code can just be deleted?

File: packages/transformers/babel/src/env.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default async function getEnvOptions(
 
 function getNeededPlugins(targets: BabelTargets): Array<PresetEnvPlugin> {
   return presetEnv(
-    {assertVersion: () => true},
+    {version: '7.13.0', assertVersion: () => true}, // version is required since https://github.com/babel/babel/pull/12934/
```
Comment (nicolo-ribaudo) [inline packages/transformers/babel/src/env.js:51]: This should be 7.12.0 (https://github.com/parcel-bundler/parcel/blob/806cc65e0e430b1120cdcf1676e2f58738326d30/packages/transformers/babel/package.json#L43), or `require("@babel/core").version`.

Ideally, the safest approach would be `{ ...require("@babel/core"), assertVersion: () => true }`.

**Comment by mischnic** [comment]
Yes, 

https://github.com/parcel-bundler/parcel/blob/806cc65e0e430b1120cdcf1676e2f58738326d30/packages/transformers/babel/src/config.js#L38-L41

means that this whole if statement could be removed because it's never reached if `!config.isSource`:

https://github.com/parcel-bundler/parcel/blob/806cc65e0e430b1120cdcf1676e2f58738326d30/packages/transformers/babel/src/env.js#L30-L41

**Comment by bemnlam** [comment]
@nicolo-ribaudo @mischnic so looks like this is a v1-only issue and what can I do if I also want to apply this change to v1? (sorry I am a newbie and I want to help 🙏)

**Comment by nicolo-ribaudo** [comment]
@bemnlam Parcel 1 is on the `master` branch, but I'd wait for the ok from a maintainer before opening a PR for Parcel 1.

**Comment by agilgur5** [comment]
@devongovett @mischnic could y'all review this change, particularly for v1? #5943 is being felt quite massively across the community as I'm sure you're seeing. I was going to write this PR myself (even though I'm only indirectly impacted) before finding it was already done here.

Currently, the only solution right now is either a `resolution` or a not-so-easy migration to the v2 beta -- this fix is considerably simpler than either.

**Comment by devongovett** [comment]
v1.12.5 has a fix. Please see https://github.com/parcel-bundler/parcel/issues/5943#issuecomment-799899064

**Pull request closed without merging**